clang/include/clang/Serialization/ASTReader.h

```diff
@@ -1188,6 +1188,11 @@ class ASTReader
   /// once recursing loading has been completed.
   llvm::SmallVector<NamedDecl *, 16> PendingOdrMergeChecks;
 
+  /// Lambdas that need to be loaded right after the function they belong to.
+  /// It is required to have the right canonical declaration for lambda class
+  /// from the same module as the function.
```

Comment:
nit: maybe it is better to expand this statement as we did in this patch. So that other readers can have a better understanding for it.